src/main/resources/application-local.conf

```
@@ -5,9 +5,11 @@ storage {
 }
 
 matrikkel {
+  useStub  = false
```

```suggestion
  useStub  = true
```

Tror jeg foretrekker å ha denne default til true, sånn at man heller må være eksplisitt med å bruke selve matrikkelen (selv om man jo også må legge til env vars).

---

Ja, jo, enig. Pushet en endring på det nå jeg.